demo/CMakeLists.txt

```diff
@@ -28,9 +30,10 @@ endif(NOT OPENGL_FOUND)
 #########################################################
 # Include Files
 #########################################################
+
 include_directories (
     ${CMAKE_CURRENT_SOURCE_DIR}
-    ../..
+    ../../.
```

Comment:
This just looks weird, mind fixing your include paths so you could use some `${PEGASUS_INCLUDE_DIRS}` here instead?